Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -117,22 +117,23 @@ public static async Task<string> GetCurrentUserNameAsync(this UITestContext cont
     public static Task SetUserRegistrationTypeAsync(
         this UITestContext context,
         UserRegistrationType type,
-        string tenant = "Default",
+        string tenant = null,
```

Comment:
Technically breaking changes, but I'd be very surprised to see somebody expecting shortcuts to go to the Default tenant when the rest of the test runs in the context of another tenant.